src/js/renderers/MultiEnumArrayControl.js

```diff
@@ -0,0 +1,114 @@
+import {
+	and,
+	hasType,
+	rankWith,
+	schemaMatches,
+	schemaSubPathMatches,
+	uiTypeIs
+} from "@jsonforms/core";
+  
+import { withJsonFormsMultiEnumProps } from "@jsonforms/react";
+import xorBy from "lodash/xorBy";
+import React from "react";
+
+import { 
+	FormTokenField,
+	__experimentalVStack as VStack,
+	Tooltip,	
+    FlexItem,
+} from '@wordpress/components';
+
+import styled from '@emotion/styled';
+
+export const GutenbergEnumArrayRenderer = ( {
+	schema,
+	visible,
+	errors,
+	path,
+	options,
+	data,
+	addItem,
+	removeItem,
+	handleChange: _handleChange,
+	description,
+	id,
+	label
+} ) => {
+
+	const HiddenLabelFormTokenFieldWrapper = styled( FlexItem )`
+		.components-form-token-field__label {
+			display: none;
+		}
+	`;
+
+	return !visible ? null : (
+		<>
+			<VStack justify="space-between">
+				<FlexItem>
+				{ description ? (
+					<Tooltip text={ description }>
+					<label htmlFor={ id }>
+						{ label }
+					</label>
+				</Tooltip>
+				) : ( 
+					<label htmlFor={ id }>
+						{ label }
+					</label> 
+				) }
+				</FlexItem>
+				<HiddenLabelFormTokenFieldWrapper>
```

Comment:
So far the `label` is shown even the label is empty, so we have to hide it using styled component